a10_octavia/common/config_options.py

```diff
@@ -310,6 +310,9 @@
     cfg.IntOpt('load_balancer_expiry_age',
                default=604800,
                help=_('Load balancer expiry age in seconds')),
+    cfg.IntOpt('stats_cleanup_interval',
```

Comment:
1. Please update design document in **Functional Requirement** and **Configuration** Section. Therefore, user can find the option in our user guide document.
2. Can we set min=60? we don't want user do this too often, since it may takes time when listener is a lot.

Reply:
Done